src/components/Display.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Display() {
+  return (
+    <div />
```

Comment:
- [ ] You need t0 use class-based components as per the project's requirement. The objective of this milestone is to practice React class components. Kindly refactor this component to a class-based component. :pray: